test/ref-list.json

```diff
@@ -2685,32 +3293,12 @@
     "CSS-UI-4-20171222",
     "css-values",
     "CSS-VALUES",
-    "css-values-20010713",
-    "CSS-VALUES-20010713",
-    "css-values-20050726",
-    "CSS-VALUES-20050726",
-    "css-values-20060919",
-    "CSS-VALUES-20060919",
-    "css-values-20110906",
-    "CSS-VALUES-20110906",
-    "css-values-20120308",
-    "CSS-VALUES-20120308",
-    "css-values-20120828",
-    "CSS-VALUES-20120828",
-    "css-values-20130404",
-    "CSS-VALUES-20130404",
-    "css-values-20130730",
-    "CSS-VALUES-20130730",
-    "css-values-20150611",
-    "CSS-VALUES-20150611",
-    "css-values-20160929",
-    "CSS-VALUES-20160929",
     "css-values-20180814",
     "CSS-VALUES-20180814",
+    "css-values-20181010",
+    "CSS-VALUES-20181010",
     "css-values-20190131",
     "CSS-VALUES-20190131",
-    "css-values-20190606",
```

Comment:
The whole point of the ref-list file is to make sure entries don't suddenly disappear. That contract is no longer fulfilled if you remove them from the ref-list. You have to automate the aliasing instead.

Reply:
Still want that one fixed ;)

Reply:
yep, on it!

Reply:
switched back "css-values-3".